TODOList.md

```diff
@@ -0,0 +1,14 @@
+# Google PACO Source Code List of TODO
```

Comment:
I am not sure what to make of this file, but, it is not our todo list. Perhaps you included it accidentally?

Incidentally, you say it will be migrating to Swift soon. Are you working on a fork of Paco and migrating it to Swift? We should talk if so, because the next version which we are currently working on will be very different from the current app and I wouldn't want you to become incompatible.